sdk/storage/storage-blob/package.json

```diff
@@ -24,14 +24,17 @@
     "build:autorest": "autorest ./swagger/README.md --typescript --package-version=12.0.1 --use=@microsoft.azure/autorest.typescript@5.0.1",
     "build:es6": "tsc -p tsconfig.json",
     "build:nodebrowser": "rollup -c 2>&1",
-    "build:js-samples": "npm run clean && npm run build:es6 && cross-env ONLY_NODE=true rollup -c 2>&1",
-    "build:ts-samples": "npm run clean && cd samples && tsc -p . ",
+    "build:samples": "npm run clean && npm run build:es6 && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run build:prep-samples",
+    "build:prep-samples": "node ../../../common/scripts/prep-samples.js && cd samples && tsc",
     "build:test": "npm run build:es6 && rollup -c rollup.test.config.js 2>&1",
     "build": "npm run build:es6 && npm run build:nodebrowser && api-extractor run --local",
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm dist-test typings temp browser/*.js* browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "clean:samples": "rimraf samples/javascript/node_modules samples/typescript/node_modules samples/typescript/dist samples/typescript/package-lock.json samples/javascript/package-lock.json",
     "extract-api": "tsc -p . && api-extractor run --local",
-    "execute:samples": "node execute-samples.js",
+    "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
+    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/samples/typescript/src/",
```

Comment:
[NIT]
`samples/typescript/dist/samples/typescript/src/` ?
`samples/typescript/dist/` is not enough?

Reply:
No, because it is not using the rollup artifact. It is actually importing the package sources as modules, so the tree in `samples/typescript/dist` after compilation looks something like this:

```dist/
  - samples/
    - typescript/
      - src/
        - basic.js
        - ... (compiled sample code)
  - src/
    - StorageClient.js
    - ... (compiled client code)
```

An alternative would be to use `ts-node` rather than compile and run the samples, like the previous script was doing, but that has its own set of problems (would need to make the *-samples scripts their own package and add ts-node as a dependency, otherwise we would be requiring global ts-node -- more than I think I can get done before leaving for the holidays).